### PR TITLE
Fix command line arg parsing; issue #797

### PR DESCRIPTION
**Pull request by jashandeep-sohi**
See #797

### CODE REVIEW

File: aiohttp/web.py [hunk 1]
```diff
@@ -349,7 +349,7 @@ def main(argv):
         type=int,
         default="8080"
     )
-    args, extra_args = arg_parser.parse_known_args(argv)
+    args, extra_args = arg_parser.parse_known_args(argv[1:])
```
Comment (asvetlov) [inline aiohttp/web.py]: Maybe better to pass `sys.argv[1:]` into `main()` call?

Comment (jashandeep-sohi) [inline aiohttp/web.py]: I don't think it really matters, either is fine. Change it if you like.

Comment (asvetlov) [inline aiohttp/web.py]: Please change. I have a guts feeling that accepting `sys.argv[1:]` is better.
Also I saw the similar approach in past a veral times.

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**